mapper/orm/src/main/java/org/hibernate/search/mapper/orm/cfg/HibernateOrmMapperSettings.java

```diff
@@ -42,24 +42,27 @@ private HibernateOrmMapperSettings() {
 	public static final String ENABLED = PREFIX + Radicals.ENABLED;
 
 	/**
-	 * Whether automatic indexing is enabled, i.e. whether changes to entities in a Hibernate ORM session
+	 * Whether ORM-triggered indexing is enabled, i.e. whether changes to entities in a Hibernate ORM session
```

Comment:
I thought the last time we discussed this we agreed on "listener-triggered indexing"? See https://hibernate.zulipchat.com/#narrow/stream/132092-hibernate-search-dev/topic/HSEARCH-4616.20new.20names.20for.20automatic.20indexing/near/365537547